### DISCUSSION TITLE
无法创建基于kerberos认证的hive数据源

### DISCUSSION THREAD

**Discussion by yuenxu** [apache/dolphinscheduler#15154] [General]
ds版本：3.2.0
hive版本：2.1.1-cdh6.3.2
报错信息：Caused by: org.apache.thrift.transport.TTransportException: Peer indicated failure: Unsupported mechanism type PLAIN

    帮忙看下是什么问题，不懂是不是配置有问题还是其他什么情况，ds 1.X 2.X 3.X，有kerberos认证的hive，数据源从未建成功过，没有kerberos认证的hive数据源倒是有建过！想知道详细一点的配置步骤，不胜感谢！

additionally，有没有ds相关的技术分享或者支持的微信群呢？

**Comment by xuhuihihi** [comment]
请问这个问题解决了吗？解决了的话，分享下，thx
